pkg/patterns/declarative/reconciler.go

```diff
@@ -124,10 +129,17 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 	if err != nil {
 		return reconcile.Result{}, err
 	}
-
-	m, err := objects.JSONManifest()
+	// run kustomize to create final manifest
```

Comment:
Can you make it so users can opt-in to kustomize? 

For a boolean example see: https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/blob/master/pkg/patterns/declarative/options.go#L149 / `WithPreserveNamespace`